Microsoft.Azure.CosmosRepository.Samples/ServiceTier/Program.cs

```diff
@@ -166,7 +192,7 @@ static async Task ServiceExampleAsync(IExampleService service)
 
             // Creating...
             Console.WriteLine("[Person] Service creating...");
-            _ = await service.AddPeopleAsync(new[] { jamesBond, adeleGoldberg });
+            _ = await service.AddPeopleAsync(new[] {jamesBond, adeleGoldberg});
```

Comment:
```suggestion
            _ = await service.AddPeopleAsync(new[] { jamesBond, adeleGoldberg });
```

Undoing some of the undesired formatting changes.

Reply:
done